GitUI/CommitInfo/CommitInfoHeader.cs

```diff
@@ -123,28 +124,30 @@ private void rtbRevisionHeader_LinkClicked(object sender, LinkClickedEventArgs e
         {
             var link = _linkFactory.ParseLink(e.LinkText);
 
-            try
+            if (Uri.TryCreate(link, UriKind.Absolute, out var uri))
             {
-                var result = new Uri(link);
-                if (result.Scheme == "gitext")
+                if (uri.Scheme == "gitext")
                 {
-                    CommandClicked?.Invoke(sender, new CommandEventArgs(result.Host, result.AbsolutePath.TrimStart('/')));
+                    CommandClicked?.Invoke(sender, new CommandEventArgs(uri.Host, uri.AbsolutePath.TrimStart('/')));
                 }
                 else
                 {
-                    using (var process = new Process
+                    using var process = new Process
                     {
                         EnableRaisingEvents = false,
-                        StartInfo = { FileName = result.AbsoluteUri }
-                    })
+                        StartInfo = { FileName = uri.AbsoluteUri }
+                    };
+
+                    try
                     {
                         process.Start();
                     }
+                    catch (Exception ex)
+                    {
+                        MessageBox.Show(this, ex.Message, new TranslationString("Error").Text, MessageBoxButtons.OK, MessageBoxIcon.Error);
```

Comment:
We don't inline `TranslationString`s.
Please move it up the top as done everywhere else.

Reply:
Please provide a screenshot of the messagebox.
I suspect we may need to add some instructions to it

Reply:
I could add that if the TranslationString is inlined, the string is not detected by the Translation app and the string is not translated.